package.json

```diff
@@ -1450,11 +1452,32 @@
             "Put the viewer tab in a new group below the current one"
           ]
         },
+        "latex-workshop.view.pdf.tab.viewColumn": {
+          "scope": "window",
+          "type": "string",
+          "default": "Beside",
+          "markdownDescription": "The view column in which to open the tab viewer, if treating it as a custom editor.",
+          "enum": [
+            "Active",
+            "Beside",
+            "One",
+            "Two",
+            "Three"
+          ],
+          "enumDescriptions": [
+            "Use the currently active editor group",
+            "Use the column next to the current editor",
+            "Use the first editor column",
+            "Use the second editor column",
+            "Use the third editor column"
+          ]
+        },
```

Comment:
At the beginning of LW development, I used a `view.pdf.tab.editorGroup` to control the viewer position. I'm not sure which one is better, but we should be consistent. I'd suggest use `view.pdf.tab.editorGroup` for now, and we may have another discussion on this after merged.

Reply:
I added a rough "translation" of `editorGroup` to `ViewColum` for now.